goose.go

```diff
@@ -3,17 +3,35 @@ package goose
 import (
 	"database/sql"
 	"fmt"
+	"regexp"
 	"strconv"
 	"sync"
 )
 
+// VerboseLevel verbose level of the goose library
+type VerboseLevel int
+
+const (
+	// VerboseOff disable the log of the executed SQL statements
+	VerboseOff VerboseLevel = iota + 1
+	// VerboseOn log the executed SQL statements
+	VerboseOn
+)
+
 var (
 	duplicateCheckOnce sync.Once
 	minVersion         = int64(0)
 	maxVersion         = int64((1 << 63) - 1)
 	timestampFormat    = "20060102150405"
+	verbose            = VerboseOff
+	reMatchSQLComments = regexp.MustCompile(`(--.*)|(((\/\*)+?[\w\W]+?(\*\/)+))`)
```

Comment:
Why not filter out just the `--` comments?

Isn't `/* */` comment/info usually useful?